examples/dh-5-example/build.gradle

```diff
@@ -0,0 +1,24 @@
+buildscript {
+    repositories {
+        maven { url 'http://distro.marklogic.com/nexus/repository/maven-releases/'}
+        maven { url 'http://distro.marklogic.com/nexus/repository/maven-snapshots/'}
+        jcenter()
+        mavenLocal()
+    }
+    dependencies {
+        classpath "com.marklogic:ml-data-hub:5.0.0"
+    }
+}
+
+plugins {
+    id 'net.saliman.properties' version '1.4.6'
+}
+
+apply plugin: "com.marklogic.ml-data-hub"
+
+task extractZip (type: Copy) {
+    from zipTree('input.zip')
+    def destDir = System.getProperty("user.dir")
+    println("input directory is extracted to: " + destDir + "/input")

```

Comment:
The "input" directory should be in a .gitignore file in this project so that the extracted files are not eligible for version control.

Reply:
Added to the .gitignore file